packages/editor-ui/src/composables/useExpressionEditor.ts

```diff
@@ -59,6 +65,7 @@ export const useExpressionEditor = ({
 	const editor = ref<EditorView>();
 	const hasFocus = ref(false);
 	const segments = ref<Segment[]>([]);
+	const selection = ref<SelectionRange>(EditorSelection.cursor(0)) as Ref<SelectionRange>;
```

Comment:
Is the assertion needed?

Reply:
This is needed, seems like the typing for vue's `ref()` modifies the type argument a bit with `UnwrapRef<T>`. If I remove the type cast, I get errors like:

```
Type 'Ref<{ readonly from: number; readonly to: number; readonly anchor: number; readonly head: number; readonly empty: boolean; readonly assoc: 0 | 1 | -1; readonly bidiLevel: number | null; ... 4 more ...; toJSON: () => any; }>' is missing the following properties from type 'SelectionRange': from, to, flags, anchor, and 9 more
```